CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.5.2 (May 27, 2022)
```

Comment:
This will be a minor version bump to v2.6.0 just because helm and kubernetes are also minor version bumps.

Reply:
You can also leave the date off for now, we'll add it when we cut the next release.